TangoRosStreamer/package.xml

```diff
@@ -9,7 +9,12 @@
   <license>Apache</license>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>tango_ros_native</build_depend>
+  <build_depend>roscpp</build_depend>
   <build_depend>rosjava_build_tools</build_depend>
```

Comment:
In the ros android [examples](https://github.com/rosjava/android_remocons/blob/kinetic/package.xml), they use build_depend.